src/lib/components/common/Tip.svelte

```diff
@@ -14,25 +14,26 @@
 <style>
   .tip {
     display: flex;
+    align-items: flex-start;
     gap: var(--gap, 1rem);
     padding: var(--padding, 1rem);
     border-radius: var(--border-radius, 1rem);
     color: var(--color, var(--gray-5, #233944));
     fill: var(--fill, var(--gray-4, #5c717c));
     background-color: var(--background-color, var(--gray-1, #f5f6f7));
     border: 1px solid var(--border-color, var(--gray-3, #99a8b3));
-
-    font-weight: var(--font-semibold);
```

Comment:
I made this change since I thought it made longer Tips more readable—I'm open to reverting, though, if anyone feels strongly the other way.

Reply:
Looks fine to me. We could pass through a style prop if we need to, but I'd wait until that need arises.